.moban.dt/gh-board-gitignore.jj2

```diff
@@ -7,5 +7,6 @@
 *.patch
 *.orig
 *.diff
+!*/src/lib/columns.js
```

Comment:
`!` is needed?

Reply:
yes, to forcefully include it despite other rules which ignore it.

Reply:
Please also add this line to `.gitignore` (see Travis failure)